netbox_prometheus_sd/api/utils.py

```diff
@@ -103,3 +103,14 @@ def extract_custom_fields(obj, labels: LabelDict):
             # Complex types are rendered as json
             else:
                 labels["custom_field_" + key.lower()] = json.dumps(value)
+
+def extract_prometheus_sd_config(obj, labels):
```

Comment:
In the example you have a ` port: 4242` defined. This seams to be ignored here. Is this intended?